src/JsonSchema/Constraints/FormatConstraint.php

```diff
@@ -197,7 +197,7 @@ protected function validateDateTime($datetime, $format)
 
     protected function validateRegex($regex)
     {
-        return false !== @preg_match('/' . $regex . '/u', '');
+        return false !== @preg_match('#' . str_replace('#', '\\#', $regex) . '#u', '')
```

Comment:
Terminating semi-colon is missing...
Besides that, please use `preg_quote` instead of of `str_replace`here...

Reply:
Do **not** use `preg_quote` here.

The `validateRegex` method is used to *validate* a regular expression by attempting to compile it. Escaping the entire regular expresson entirely defeats the point of that logic, and would result in any string being considered a valid expression.

Reply:
But the semicolon has been indeed accidentally removed. I've used github code editor on smartphone ;)

Reply:
Fixed the semicolon

Reply:
@erayd Thanks for pointing it out